Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -73,7 +78,7 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
         case .httpPostRequestError(let error):
             return "HTTP POST request failed with \(error)."
         case .invalidURL(let error):
```

Comment:
Do we want a separate enum case for differentiating approval URL & redirect URL related errors?

Reply:
🤔 we could, we can also consider a new error case just for app switch and move the use of this error in this PR and in `launchPayPalApp` with `invalidAppSwitchURL`. That would allow us to revert the changes to `invalidURL` in this branch and instead use a new case for app switch related URL errors. What do you think?

Reply:
Resolved, a new enum case `appSwitchReturnURLPathInvalid` was added